### PR TITLE
init commit -- poc version + documentation.

### PR DESCRIPTION
**Pull request by tybalex**
This commit simply moves the code from MAD pod to this official opni repo.

### CODE REVIEW

**Comment by tybalex** [comment]
To update:
1. readme to setup prometheus & Grafana
2. basic docstrings for code.

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
...
+#### Prerequisites:
+Kubernetes cluster running with Prometheus and Grafana and Opni installed.
+Rancher user can install Prometheus and Grafana by [enable Rancher monitoring](https://rancher.com/docs/rancher/v2.5/en/monitoring-alerting/guides/enable-monitoring/).
+
+
+#### Build and Run
+To build the metric-anomaly-detection Docker image and then push the image:
+```
+docker build -t opni-metric-forecasting-service ./
+docker tag opni-metric-forecasting-service-service [ACCOUNT_NAME]/opni-metric-forecasting-service
```
Comment (AmartC) [inline README.md]: Should be:
```
docker tag opni-metric-forecasting-service [ACCOUNT_NAME]/opni-metric-forecasting-service
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM python:3.8-slim
```
Comment (AmartC) [inline Dockerfile]: Rancher base image should be used.

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+pandas==1.2.3
```
Comment (AmartC) [inline requirements.txt:1]: The requirements.txt file with the main Opni repository must be modified to have these additional libraries.
https://github.com/rancher/opni/blob/main/images/python/requirements.txt

File: requirements.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+prometheus_api_client==0.4.2
+numpy
+statsmodels==0.12.2
+opni-nats==0.0.0.2
```
Comment (AmartC) [inline requirements.txt]: Currently should be using opni-nats version 0.0.0.4

File: requirements.txt (hunk 3 above)
Comment (tybalex) [reply requirements.txt:1]: Thanks to @AmartC  this has been addressed in this PR: https://github.com/rancher/opni/pull/104

File: README.md (hunk 1 above)
Comment (tybalex) [reply README.md]: fixed.

File: Dockerfile (hunk 2 above)
Comment (tybalex) [reply Dockerfile]: updated with rancher's base image

File: requirements.txt (hunk 4 above)
Comment (tybalex) [reply requirements.txt]: updated to latest version

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,38 @@
+SEVERITIES = HIGH,CRITICAL
+
+ifeq ($(ARCH),)
+ARCH=$(shell go env GOARCH)
+endif
+
+ORG ?= rancher
+TAG ?= v0
+REPO=opni-metric-anomaly-detector
```
Comment (dbason) [inline Makefile]: Can we just call this opni-metrics-service? This naming would match the other services

File: README.md [hunk 6]
```diff
@@ -1 +1,56 @@
...
+## Installation
+#### Prerequisites:
+Kubernetes cluster running with Prometheus and Grafana and Opni installed.
+Rancher user can install Prometheus and Grafana by [enable Rancher monitoring](https://rancher.com/docs/rancher/v2.5/en/monitoring-alerting/guides/enable-monitoring/).
+
+
+#### Build and Run
+To build the metric-anomaly-detection Docker image and then push the image:
+```
+docker build -t opni-metric-anomaly-detector ./
```
Comment (dbason) [inline README.md]: As per previous comment about naming

**Review by dbason — CHANGES_REQUESTED** [review]

File: Makefile (hunk 5 above)
Comment (tybalex) [reply Makefile]: updated

File: README.md (hunk 6 above)
Comment (tybalex) [reply README.md]: updated

**Review by dbason — APPROVED** [review]

**Review by AmartC — APPROVED** [review]
LGTM

**Pull request merged**